### PR TITLE
Master add python meh support

### PR DESCRIPTION
**Pull request by jkonecny12**
Add https://github.com/rhinstaller/python-meh project support.

### CODE REVIEW

File: anaconda_updates/anaconda_updates/releases/__init__.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def __init__(self, branch_type,
                  pykickstart_args=[],
                  simpleline_args=[],
                  dasbus_args=[],
+                 meh_args=[],
```
Comment (VladimirSlavik) [inline anaconda_updates/anaconda_updates/releases/__init__.py]: Maybe this could use a refactor too? `proj_args={"meh": "blah"}` etc.

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: anaconda_updates/anaconda_updates/releases/__init__.py (hunk 1 above)
Comment (jkonecny12) [reply anaconda_updates/anaconda_updates/releases/__init__.py]: Fixed by a complete removal of these. They are not used anyway.

Comment (jkonecny12) [reply anaconda_updates/anaconda_updates/releases/__init__.py]: Quote from IRC:
```
<sbueno> so maybe saying something like: "Encrypt automatically created mount points by default:"
```

**Review by VladimirSlavik — APPROVED** [review]
Even better! Thanks!

**Pull request merged**